src/main.rs

```diff
@@ -53,13 +53,12 @@ fn main() {
       }
     }
   }
-  let expression = match expression_opt {
-    Some(expression) => expression,
-    None => {
-      print_help();
-      exit(0);
-    },
+  #[allow(clippy::option_if_let_else)]
+  let expression = if let Some(expression) = expression_opt { expression } else {
+    print_help();
+    exit(0);
```

Comment:
`if let` isn't very readable, so I avoid it whenever I can

Reply:
reverting back...